converter/complete.go

```diff
@@ -248,7 +248,7 @@ func listCandidatesFromScope(s *types.Scope, pos token.Pos, prefix string, candi
 		return
 	}
 	for _, name := range s.Names() {
-		if !strings.HasPrefix(name, prefix) {
+		if !strings.HasPrefix(strings.ToLower(name), prefix) {
```

Comment:
Thank you for catching this. Can you send a pull request only for this fix?
(You can create a bug entry for this issue if you want).
I can not accept you change in `func complete` but I want to accept this change.

Reply:
Sure, will do!